core/transformer/src/IModelTransformer.ts

```diff
@@ -627,24 +800,24 @@ export class IModelTransformer extends IModelExportHandler {
 
   /** Import elements that were deferred in a prior pass.
    * @note This method is called from [[processChanges]] and [[processAll]], so it only needs to be called directly when processing a subset of an iModel.
+   * @deprecated
```

Comment:
I am not concerned that the API will silently stop working. I think I will keep it deprecated to give anyone that is using it time to remove it. But I will modify the note

Reply:
Anyone who is currently using it will need to update their code anyway, because it no longer does anything. Doesn't deprecating it just silently break their code, whereas deleting it loudly notifies them their code is broken?

Reply:
it is only supposed to be called manually after a partial transformation. Now users doing that will see no change since the transformation never deferred elements, so calling an empty function to finish processing will do nothing and their partial transformation will still have completed correctly previously.

Anyone overriding it themselves would probably only do such a thing for logging, which will continue to work. I am trying to think of any ways they could have overridden it that I would be breaking but I can't think of any... the deferring was an implementation detail and e.g. `skipElement` should never have been exposed, and any attempt to manually handle deferred elements in an override will find that no elements were deferred and should therefore not have any problems

Reply:
Got it, thanks. Please edit NextVersion.md to emphasize the deprecations are of `beta` APIs.

Reply:
done